.github/workflows/ci.yml

```diff
@@ -28,7 +32,11 @@ jobs:
       - name: cache node_modules
         uses: actions/cache@v1
         with:
-          path: node_modules
+          path: |
```

Comment:
We should update cache to: https://github.com/actions/cache/blob/main/examples.md#node---yarn

By caching global-dir, we don't need to workaround playwright